<launch>
  <!-- Launch file for velodyne16 VLP-16 LiDAR -->

    <arg name="rviz" default="true" />

    <rosparam command="load" file="$(find sr_lio)/config/ulhk_1.yaml" />

    <param name="debug_output" type="bool" value="0"/>
    <param name="output_path" type="string" value="$(find sr_lio)/output"/>
    <node pkg="sr_lio" type="lio_optimization" name="lio_optimization" output="screen" /> 

    <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find sr_lio)/rviz_cfg/visualization.rviz" />
    </group>

</launch>
